framework/src/wifi_manager/wifi_manager_api.c

```diff
@@ -210,6 +210,26 @@ wifi_manager_result_e wifi_manager_scan_ap(wifi_manager_scan_config_s *config)
 	RETURN_RESULT(wifimgr_post_message(&msg), msg);
 }
 
+wifi_manager_result_e wifi_manager_scan_multi_aps(wifi_manager_scan_multi_configs_s *configs)
+{
+	NET_LOGI(TAG, "--> %s %d\n", __FUNCTION__, __LINE__);
+	if (configs) {
```

Comment:
May I ask if we should do a check here on whether config is NULL?
RTK's implementation of scanning multiple AP API requires config to be passed in, if it is NULL the API will fail.

Reply:
@jwei5 My opinion is that is better to operate in the same way as single AP scan.
In the future, the single AP scan command may be completely replaced to the multi scan.